### PR TITLE
Linebreaks in version.txt no longer breaks build

### PR DESCRIPTION
**Pull request by cmelchior**
We can now edit the version.txt from GitHub. Yay

@emanuelez @kneth

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -17,7 +17,7 @@ allprojects {
   apply plugin: 'android-sdk-manager'
 
   group = 'io.realm'
-  version = new File("${rootDir}/version.txt").text
+  version = new File("${rootDir}/version.txt").text.replaceAll("\\r|\\n","")
```
Comment (emanuelez) [inline build.gradle]: How about using `trim()`?

Comment (cmelchior) [inline build.gradle]: I tried initially and it failed. At first I thought it was perhaps trim() wasn't available: http://www.groovy-lang.org/gdk.html, but after remembering to use it in all 3 projects, it does actually work, so much better than custom regexp.

**Comment by kneth** [comment]
:+1:

**Comment by emanuelez** [comment]
:+1:

**Pull request merged**